lib/ui/onboarding/import_account/import_words/import_account_by_words_view.dart

```diff
@@ -95,14 +95,14 @@ class ImportAccountByWordsView extends StatelessWidget {
                         child: HyphaProgressIndicator(),
                       );
                     } else if (state.showNoAccountsFound) {
-                      return Padding(
-                        padding: const EdgeInsets.only(top: 16),
+                      return const Padding(
+                        padding: EdgeInsets.only(top: 16),
                         child: Row(
                           mainAxisAlignment: MainAxisAlignment.center,
                           children: [
-                            const Icon(Icons.info_outline, color: HyphaColors.error),
-                            const SizedBox(width: 8),
```

Comment:
has the "const" thing been removed in Flutter 3? Is it doing it automatically now? that would be great...

Reply:
That would be indeed great. But i dont think so. 
I think the compiler got a bit smarter and realized there is a constant parent up top, therefore the children dont need to be marked as constant